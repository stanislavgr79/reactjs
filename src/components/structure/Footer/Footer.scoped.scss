.footer {
    background-color: #424242;
    padding: 15px;
    height: 65px;
    p {
        margin: auto;
        text-align: center;
    }
}