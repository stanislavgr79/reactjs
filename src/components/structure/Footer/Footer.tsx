import React from 'react';
import SiteName from '@components/atom/SiteName';
import './Footer.scoped.scss';

export default function Footer(): JSX.Element {
  return (
    <div className="footer">
      <SiteName />
    </div>
  );
}
