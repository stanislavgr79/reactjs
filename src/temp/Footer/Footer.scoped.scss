.footer {
    background-color: #424242;
    padding: 15px;
    height: 50px;
    p {
        margin: auto;
        text-align: center;
    }
}