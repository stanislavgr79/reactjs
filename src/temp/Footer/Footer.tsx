import React from 'react';
import './Footer.scoped.scss';

const Footer = (props: { children?: React.ReactNode }): JSX.Element => (
  <div className="footer">{props.children}</div>
);

export default Footer;
