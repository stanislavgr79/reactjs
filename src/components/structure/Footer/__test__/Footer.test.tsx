import React from 'react';
import { render } from '@testing-library/react';

import Footer from '../Footer';

jest.mock('../../../atom/SiteName', () => {
  return {
    __esModule: true,
    default: () => {
      return <p className="site_name"></p>;
    },
  };
});

describe('Footer', () => {
  test('snapshot renders', () => {
    const tree = render(<Footer />);

    expect(tree.baseElement).toMatchSnapshot();
    expect(tree.baseElement).toContainHTML('<div class="footer">');
  });
});
