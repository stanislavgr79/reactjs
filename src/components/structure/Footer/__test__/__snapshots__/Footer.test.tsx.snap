// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot renders 1`] = `
<body>
  <div>
    <div
      class="footer"
    >
      <p
        class="site_name"
      />
    </div>
  </div>
</body>
`;
